superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/ag-grid/GridHeader/GridHeader.tsx

```diff
@@ -0,0 +1,11 @@
+
+const GridHeader = () => {
+
```

Comment:
Can we remove a few blank lines here?

Reply:
File was included accidentally it looks like, removing it